README.md

```diff
@@ -33,7 +33,7 @@ See the `docs/` directory for more documentation.
 #### AWS
 
 Amazon AWS is used by the app to store images as well as transferring db backups between environments.
-Get an invitation to the AWS account and configure an API key and secret. Install the [aws cli](https://docs.aws.amazon.com/cli/latest/userguide/installing.html) and [configure it](https://docs.aws.amazon.com/cli/latest/userguide/cli-chap-getting-started.html).
+Get an invitation to the AWS account and configure an API key and secret. Install the [aws cli](https://docs.aws.amazon.com/cli/latest/userguide/installing.html)(already included in Brewfile) and [configure it](https://docs.aws.amazon.com/cli/latest/userguide/cli-chap-getting-started.html).
```

Comment:
Minor, but missing a space before `(already...`.

Reply:
AWS's own documentation does not list Homebrew as an installation method, how's it working for you?

Reply:
the [brew tap](https://github.com/Homebrew/homebrew-core/blob/master/Formula/awscli.rb) is pretty straightforward - it adds cli completion though which is handy.